string camera_name
---
bool success
